packages/mdl-radio/mdl-radio.scss

```diff
@@ -0,0 +1,108 @@
+$mdl-radio-size: 20px !default;
+$mdl-radio-foreground-color: #333 !default;
+$mdl-radio-accent-color: rgba(255,82,82,0.87) !default;
+$mdl-radio-disabled-color: gray !default;
+$mdl-radio-timing-fn: ease 280ms !default;
+
+// Top-level host container.
+.mdl-radio {
+  color: $mdl-radio-foreground-color;
+  cursor: pointer;
+  display: inline-block;
+  transition: color $mdl-radio-timing-fn;
+}
+
+// Container for radio circles and ripple.
+.mdl-radio__container {
```

Comment:
I think your approach is better and we should probably standardize on it. For most components, thinking about the component in terms of top-level behavior states (checked, disabled, etc.) and then nesting modifications seems more intuitive than elements containing their modifiers.